ports/esp32/network_wlan.c

```diff
@@ -508,6 +508,24 @@ STATIC mp_obj_t network_wlan_config(size_t n_args, const mp_obj_t *args, mp_map_
                         esp_exceptions(esp_wifi_set_protocol(self->if_id, mp_obj_get_int(kwargs->table[i].value)));
                         break;
                     }
+                    case MP_QSTR_pm: {
+                        mp_obj_t arg = kwargs->table[i].value;
+                        mp_int_t pm_mode;
+                        if (!mp_obj_get_int_maybe(arg, &pm_mode)) {
```

Comment:
I suggest to remove this capability for now, and we can later work out how to make it compatible across WiFi drivers.